transit_odp/timetables/transformers.py

```diff
@@ -181,3 +283,81 @@ def sync_stop_points(self, stop_points, provisional_stops):
         # Return the subselection of stop points seen in the doc (useful when
         # processing large zip files)
         return stop_point_cache.reindex(sorted(stop_point_refs))
+
+    def sync_flexible_zone(self, flexible_stop_points):
+        filtered_stop_points = flexible_stop_points[
+            (flexible_stop_points["bus_stop_type"] == "flexible")
+            & (~flexible_stop_points["naptan_id"].isna())
```

Comment:
Flexible stops which are not in stop point and not n flexible zone will be removed. We need this data even though its not in naptan and flexible zone